framework/encode/capture_manager.cpp

```diff
@@ -392,6 +393,11 @@ bool CommonCaptureManager::Initialize(format::ApiFamilyId                   api_
             GFXRECON_ASSERT((trim_boundary_ == CaptureSettings::TrimBoundary::kFrames) ||
                             (trim_boundary_ == CaptureSettings::TrimBoundary::kQueueSubmits));
 
+            if (trim_boundary_ == CaptureSettings::TrimBoundary::kQueueSubmits)
+            {
+                GFXRECON_LOG_WARNING("Trimming queue submits will be changed to 0-based.");
```

Comment:
Let's use this message:
"Capture has enabled the GFXRECON_CAPTURE_QUEUE_SUBMITS option. This option currently uses 1-based indexing to identify the queue submit range. In the future it will be switched to 0-based indexing."

Reply:
fixed